src/main/java/org/gnieh/logback/config/ConfigConfigurator.java

```diff
@@ -124,22 +128,29 @@ public void configure(LoggerContext loggerContext) {
 				Object value = entry.getValue().unwrapped();
 				if (value instanceof String) {
 					propertySetter.setProperty(propertyName, (String) value);
+				} else if (value instanceof Map) {
```

Comment:
I really don’t like the way it is done but I am wondering whether it can be done otherwise… Did you take this code from the default configurator?